ubr/s3.py

```diff
@@ -99,8 +99,14 @@ def parse_s3_project_files(bucket, project):
     return parse_path_list(s3_project_files(bucket, project))[project]
 """
 
-def filterasf(file_list, project, host, filename):
-    import re
+def filterasf(file_list, project, host, target=None, filename=''):
```

Comment:
`filter-as-f` - I'm mystified what `f` originally was. perhaps it *was* short for `filter-asfasdf` , code archaeologists will be pondering this for years to come. I'll give it a new name.